docs/integrations.md

```diff
@@ -25,21 +25,12 @@ On each scan that has at least one finding, you will receive one email from Semg
 ### Pull request comments
 
 !!! info
-    This feature is currently only available for GitHub. There’s no need to create `secrets.GITHUB_TOKEN` yourself because it’s automatically set by GitHub. It only needs to be passed to Semgrep CI via the workflow file.
+    This feature is currently only available for GitHub.
```

Comment:
Wasn't sure if I should reword this or remove it. As I understand it, we only leave inline PR comments in GitHub, but it should work now for other CI providers.

Reply:
I think what you've done here makes sense